Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -490,10 +490,13 @@ private void EmitTradeTick(Messages.Matched message)
         /// </summary>
         public override void Subscribe(IEnumerable<Symbol> symbols)
         {
-
             foreach (var item in symbols)
             {
-                if (item.Value.Contains("UNIVERSE")) continue;
+                if (item.Value.Contains("UNIVERSE") ||
+                    item.SecurityType != SecurityType.Forex && item.SecurityType != SecurityType.Crypto)
```

Comment:
I'm beginning t think we need a more deterministic way of defining what is a universe symbol. Consider a user defined universe where they give it whatever symbol they desire.

I'm not recommending a change for this PR, but just planting the seed for future thought :)